app/views/posts/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="border-4 p-2">
+  <h4><%= @post.title %></h4>
+  <p><%= @post.text %></p>
+  <div class="flex gap-2 justify-end">
+    <div class="flex gap-2">
+      <p>Comments:</p>
+      <p><%= @post.comments_count %></p>
+    </div>
+    <div class="flex gap-2">
+      <p>Likes:</p>
+      <p><%= @post.likes_count %></p>
+    </div>
+  </div>
+</div>
+
+<ul class="w-full m-0 p-0 border-4">
+  <% @post.comments.each do |comment|%>
+    <%= render partial: "partials/comments", locals: {comment: comment}%>
+  <% end %>
+</ul>
+
+<div class="flex justify-center m-4">
+<%= link_to 'Back to all posts', user_posts_path(@post.author)%>
+</div>
```

Comment:
**OPTIONAL** - Please note that according to the project requirements, you are not required to have any logic in your view files, this would be done in a later milestone. I suggest having only a placeholder in this file 👍 👍